### PR TITLE
Feature/43 workflow

### PR DESCRIPTION
**Pull request by Abibibi**

### CODE REVIEW

File: src/components/App/App.scss [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+    overflow-x: hidden;
 }
 
 #root {
+    background-color: #323742;
     & .btn {
         margin: 1vh 0 2rem;
         background-color: #468ea0;
-        padding: .375rem 1.5rem;
+        padding: .375rem 1.5rem;/*  */
```
Comment (HugoGresse) [inline src/components/App/App.scss]:
```suggestion
        padding: .375rem 1.5rem;
```

File: src/components/BlogArticle/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    useEffect(() => {
+        document.title = `${title} - Blog - Instant Visio`
+        blogContent.current.innerHTML = content
+    }, [])
+
+    return(
+        <BlogArticleStyled>
+            <Link to={`/blog/${slug}`} className="article-title"><h2 className="default-title">{title}</h2></Link>
+            <p className="article-dateAuthor">Le {date}, par {author}</p>
```
Comment (HugoGresse) [inline src/components/BlogArticle/index.js]: not i18n compatible, e could use Luxon here so support date internationalization through the browser

File: src/components/Footer/index.js [hunk 3]
```diff
@@ -11,13 +11,35 @@ const Footer = () => {
...
                     >
                         {t('footer.legal-mentions')}
                     </Link>
                 </li>
+                <li className="footer-link">
+                    <Link
+                        to="/donnees-personnelles"
+                        className="footer-link-content"
+                    >
+                        Données personnelles
```
Comment (HugoGresse) [inline src/components/Footer/index.js]: Missing translation

File: src/components/Footer/index.js [hunk 4]
```diff
@@ -11,13 +11,35 @@ const Footer = () => {
...
+                    </Link>
+                </li>
+                <li className="footer-link">
+                    <Link
+                        to="/blog"
+                        className="footer-link-content"
+                    >
+                        Blog
```
Comment (HugoGresse) [inline src/components/Footer/index.js]: Missing translation

File: src/components/Footer/index.js [hunk 5]
```diff
@@ -11,13 +11,35 @@ const Footer = () => {
...
+                    </Link>
+                </li>
+                <li className="footer-link">
+                    <Link
+                        to="/credits"
+                        className="footer-link-content"
+                    >
+                        Crédits
```
Comment (HugoGresse) [inline src/components/Footer/index.js]: Missing translation

File: src/pages/Blog/index.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+                    slug,
+                    title,
+                    date,
+                    author,
+                    content
+                }) => {
+
+                    // either all articles are displayed,
+                    // or the article matching the URL
+                    if (article === slug || !article) {
```
Comment (HugoGresse) [inline src/pages/Blog/index.js]: neat 😅

File: src/pages/Credits/index.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
+import React, {useEffect} from 'react'
+
+import DefaultLayout from '../../layout/Default'
+import CreditsStyled from './Credits'
+
+
+const Credits = () => { 
+    useEffect(() => {
+        document.title = 'Crédits - Instant Visio'
```
Comment (HugoGresse) [inline src/pages/Credits/index.js]: Missing translation

File: src/pages/Home/Description.js [hunk 8]
```diff
@@ -85,19 +98,23 @@ const Information = styled.div`
...
+        <div>
+            <Information>
+                <p>{t('information.title')}</p>
+                <List>
+                    <span>{t('information.steps.1')}</span>
+                    <span>{t('information.steps.2')}</span>
+                    <span>{t('information.steps.3')}</span>
+                </List>
+            </Information>
+            <p className="details">Après avoir soumis le formulaire, nous vous conseillons de placer votre appareil en mode paysage. Vous verrez d'autant mieux votre proche.</p>
```
Comment (HugoGresse) [inline src/pages/Home/Description.js]: Missing translation

File: src/pages/Home/Description.js [hunk 9]
```diff
@@ -85,19 +98,23 @@ const Information = styled.div`
...
+            <p className="details">Une fois sur la vidéo, vous avez accès au lien de l'appel dans la barre d'adresses. Vous pouvez la communiquer à d'autres proches, de votre côté. Nous vous recommandons de vous limiter à 6 participants par appel, pour une expérience optimale.</p>
```
Comment (HugoGresse) [inline src/pages/Home/Description.js]: Missing translation

File: src/pages/Home/index.js [hunk 10]
```diff
@@ -43,6 +44,8 @@ export default function Home(){
                             />
                     }
                 </div>
+                <p className="cnil">Les champs marqués d'une astérisque sont obligatoires.</p>
```
Comment (HugoGresse) [inline src/pages/Home/index.js]: Missing translation

File: src/pages/Home/index.js [hunk 11]
```diff
@@ -43,6 +44,8 @@ export default function Home(){
...
+                <p className="cnil">Le responsable de traitement, Stéphane Luçon, s'assure du traitement des données recueillies pour effectuer l'envoi du SMS ou de l'e-mail au correspondant. Suite à l'envoi, ces données sont effacées au bout d'un jour. Pour en savoir plus sur la gestion des données personnelles et pour exercer vos droits, veuillez vous reporter à la page <Link to="/donnees-personnelles">Données personnelles</Link>.</p>
```
Comment (HugoGresse) [inline src/pages/Home/index.js]: Missing translation

File: src/components/App/App.scss (hunk 1 above)
Comment (Abibibi) [reply src/components/App/App.scss]: ok, deleted the comment

File: src/components/BlogArticle/index.js (hunk 2 above)
Comment (Abibibi) [reply src/components/BlogArticle/index.js]: ok, used Luxon

File: src/components/Footer/index.js (hunk 3 above)
Comment (Abibibi) [reply src/components/Footer/index.js]: done

File: src/components/Footer/index.js (hunk 4 above)
Comment (Abibibi) [reply src/components/Footer/index.js]: done

File: src/components/Footer/index.js (hunk 5 above)
Comment (Abibibi) [reply src/components/Footer/index.js]: done

File: src/pages/Blog/index.js (hunk 6 above)
Comment (Abibibi) [reply src/pages/Blog/index.js]: cool :)

File: src/pages/Credits/index.js (hunk 7 above)
Comment (ntchou) [reply src/pages/Credits/index.js]: You can move useEffect to a customHook. Call this customhook in differents layouts. Inject title prop to layout

Comment (Abibibi) [reply src/pages/Credits/index.js]: done

File: src/components/BlogArticle/index.js [hunk 12]
```diff
@@ -21,7 +22,7 @@ const BlogArticle = ({
     return(
         <BlogArticleStyled>
             <Link to={`/blog/${slug}`} className="article-title"><h2 className="default-title">{title}</h2></Link>
-            <p className="article-dateAuthor">Le {date}, par {author}</p>
+            <p className="article-dateAuthor">Le {DateTime.fromISO(date).toLocaleString(DateTime.DATE_FULL)}, par {author}</p>
```
Comment (HugoGresse) [inline src/components/BlogArticle/index.js]: 👍

File: src/layout/Columns/index.js [hunk 13]
```diff
@@ -16,26 +17,33 @@ const Left = styled.div`
...
     background: ${({theme}) => theme.color.grey};
     width: 100%;
     padding: ${({theme}) => theme.spacing.M};
-    ${SCREEN.DESKTOP_AND_TABLET} {
+    ${SCREEN.DESKTOP} {
         width: 45%;
         padding: 7rem;
     }
+
+    .cnil {
```
Comment (ntchou) [inline src/layout/Columns/index.js]: it could be better to move it in Home page to keep the layout the most pure as possible

File: yarn.lock [hunk 14]
```diff
@@ -910,52 +878,14 @@
...
-
-"@emotion/styled@^10.0.27":
-  version "10.0.27"
-  resolved "https://registry.yarnpkg.com/@emotion/styled/-/styled-10.0.27.tgz#12cb67e91f7ad7431e1875b1d83a94b814133eaf"
-  dependencies:
-    "@emotion/styled-base" "^10.0.27"
-    babel-plugin-emotion "^10.0.27"
-
-"@emotion/stylis@0.8.5", "@emotion/stylis@^0.8.4":
+"@emotion/stylis@^0.8.4":
```
Comment (ntchou) [inline yarn.lock]: we can delete everything related to emotion

File: src/components/BlogArticle/index.js [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+        }
+    }, [])
+
+    return(
+        <BlogArticleStyled>
+            <Link to={`/blog/${slug}`} className="article-title"><h2 className="default-title">{title}</h2></Link>
+            <p className="article-dateAuthor">{DateTime.fromISO(date).toLocaleString(DateTime.DATE_FULL)}, par {authors.join(` ${authorsAnd} `)}</p>
+            <ReactMarkdown source={content} />
+            <div className="article-forSharing">
+                Partager sur :
```
Comment (ntchou) [inline src/components/BlogArticle/index.js]: Missing translation ;)

File: src/components/BlogArticle/index.js [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+    slug,
+    title,
+    pageTitle,
+    date,
+    authors,
+    authorsAnd,
+    content
+}) => {
+
+    useEffect(() => {
```
Comment (ntchou) [inline src/components/BlogArticle/index.js]: you can use hook `useDocumentTitle`

File: src/components/Form/form.css [hunk 17]
```diff
@@ -1,3 +1,16 @@
+#root .form-group {
```
Comment (ntchou) [inline src/components/Form/form.css]: we can delete file. Not used

File: src/pages/PersonalData/index.js [hunk 18]
```diff
@@ -0,0 +1,19 @@
...
+import { useTranslation } from 'react-i18next'
+import ReactMarkdown from 'react-remarkable'
+
+import DefaultLayout from '../../layout/Default'
+
+const PersonalData = () => {
+    const {t} = useTranslation('personal-data')
+
+    return (
+        <>
```
Comment (ntchou) [inline src/pages/PersonalData/index.js]: not needed

File: src/components/Logo/index.js [hunk 19]
```diff
@@ -8,5 +10,15 @@ import useDetectMobile from '../../hooks/useDetectMobile'
 export default function Logo(){
     const {t} = useTranslation()
     const isMobile = useDetectMobile()
-    return (<img src={isMobile ? logoMobile : logo} alt={t('logo.alt')} />)
+    let logoToChoose = ''
+
+    if (isMobile) {
+        logoToChoose = logoMobile
+    } else if (window.location.pathname === '/' && !isMobile) {
+        logoToChoose = homeLogo
```
Comment (ntchou) [inline src/components/Logo/index.js]: we can use same `homeLogo.svg` for all the pages. it 's more optimized than `logo.svg`

File: src/pages/PersonalData/index.js (hunk 18 above)
Comment (ntchou) [inline src/pages/PersonalData/index.js]: not needed

File: src/pages/Home/index.js [hunk 20]
```diff
@@ -29,7 +33,7 @@ export default function Home(){
             })
     }
     return (
-        <ColumnsLayout>
+        <ColumnsLayout title={`${t('page-title')} - Instant Visio`}>
             <Description />
             <>
```
Comment (ntchou) [inline src/pages/Home/index.js]: not needed

File: src/pages/PersonalData/index.js (hunk 18 above)
Comment (Abibibi) [reply src/pages/PersonalData/index.js]: Please, what is not needed here?

File: src/pages/Home/index.js (hunk 20 above)
Comment (Abibibi) [reply src/pages/Home/index.js]: If I remove it, and I go from Blog page back to Home page, document title is still "Blog - Instant Visio"

File: src/components/Logo/index.js (hunk 19 above)
Comment (Abibibi) [reply src/components/Logo/index.js]: homeLogo.svg comes with a margin, so it is used in other pages, logo is indented to the right and the symmetry effect with "Joignez un proche en visio..." created with justify-content: space-between is lost.

File: src/components/BlogArticle/index.js (hunk 15 above)
Comment (Abibibi) [reply src/components/BlogArticle/index.js]: ah yes! Done!

File: src/layout/Columns/index.js (hunk 13 above)
Comment (Abibibi) [reply src/layout/Columns/index.js]: done

**Pull request merged**